src/core/lib/security/credentials/tls/grpc_tls_credentials_options.cc

```diff
@@ -87,20 +87,23 @@ void grpc_tls_server_authorization_check_config::Cancel(
 /** -- Wrapper APIs declared in grpc_security.h -- **/
 
 grpc_tls_credentials_options* grpc_tls_credentials_options_create() {
+  grpc_core::ExecCtx exec_ctx;
   return new grpc_tls_credentials_options();
 }
 
 void grpc_tls_credentials_options_set_cert_request_type(
     grpc_tls_credentials_options* options,
     grpc_ssl_client_certificate_request_type type) {
   GPR_ASSERT(options != nullptr);
+  grpc_core::ExecCtx exec_ctx;
```

Comment:
This one probably isn't needed, because there's nothing here that could invoke callbacks.